PythonCodeConventions.md

```diff
@@ -49,6 +57,48 @@ y = fib(2.5)
 In this case, mypy will inform that the call using the floating point value is not valid:
 __mypytest.py:10: error: Argument 1 to "fib" has incompatible type "float"; expected "int"__
 
+The following construct is _only_ allowed in case of unavoidable circular dependencies, otherwise regular import statements MUST be used.
+``` python
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from [package] import [class]
+```
+Line breaking
+---
+Indentation of function definitions and function calls that exceed the soft line length (PEP8: 80, Ultimaker: 120 characters) is described in PEP8 https://www.python.org/dev/peps/pep-0008/#indentation. From the suggested set, the following SHOULD be applied:
```

Comment:
Don't hide the 120 character specification inside a paragraph but make it standout on its own line.

Reply:
It has his own line in the code_conventions.md, stated here as a quick reference. Can either stay this way or be removed entirely.